project/AppDependencies.scala

```diff
@@ -7,7 +7,7 @@ object AppDependencies {
 
   val compile = Seq(
     "uk.gov.hmrc"                %% "bootstrap-frontend-play-28" % "5.3.0",
-    "uk.gov.hmrc"                %% "play-frontend-hmrc"         % "0.88.0-play-28",
+    "uk.gov.hmrc"                %% "play-frontend-hmrc"         % "1.11.0-play-28",
```

Comment:
This is not the latest version: update to 1.14.0-play-28

Reply:
Changed.